Automattic/jetpack
# Block Icons: change foreground color on WPCOM

#### Changes proposed in this Pull Request:
The `getIconColor` function sets the Jetpack Green for non-WPCOM sites and uses the default Gutenberg color ( `#1e1e1e` ) on WPCOM sites. This causes the icon not to change color depending on the location. By returning `null` this unsets the foreground setting and allows it to change colors.

Loosely mentioned in this issue Automattic/wp-calypso#48959

#### Before

<img width="334" alt="Markup 2021-10-07 at 12 38 42" src="https://user-images.githubusercontent.com/33258733/136374756-ab5675a7-0ba4-4679-a97a-bb7e93ee06f0.png">

<img width="401" alt="Markup 2021-10-07 at 13 25 32" src="https://user-images.githubusercontent.com/33258733/136375254-ce365081-d76d-4c54-a156-b0b13ce85706.png">

<img width="274" alt="Markup 2021-10-07 at 13 26 53" src="https://user-images.githubusercontent.com/33258733/136375459-d6c7889a-cb63-4743-9270-43b9c5ba3a0f.png">

#### After

<img width="359" alt="Markup 2021-10-07 at 13 24 35" src="https://user-images.githubusercontent.com/33258733/136375087-437171cd-cebd-407a-8a1b-64eb45f37237.png">

<img width="471" alt="Markup 2021-10-07 at 13 25 48" src="https://user-images.githubusercontent.com/33258733/136375273-8921578e-6e6f-490f-9955-6f37f34f43e0.png">

<img width="369" alt="Markup 2021-10-07 at 13 27 16" src="https://user-images.githubusercontent.com/33258733/136375499-d2225514-db17-476e-b00f-fc9651aeb0e6.png">

#### Does this pull request change what data or activity we track or use?


Nope

#### Testing instructions:
* Create a new post
* Add a core block and a Jetpack block
* Look at all instances of the icon. They should change color to match the text next to it rather than staying black.
